api/python/t4/packages.py

```diff
@@ -48,10 +47,10 @@ def get_package_registry(path=''):
     if path.startswith('s3://'):
         bucket = path[5:].partition('/')[0]
         return "s3://{}/.quilt".format(bucket)
-    else:
-        return get_local_package_registry().as_uri()
+    # default to local registry
```

Comment:
Nit: Let's try to make comments close to sentences, capital first letter and punctuation at the end.